package.json

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "license": "MIT",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "publisher": "XebiaLabs",
   "preview": true,
   "bugs": "http://xebialabs.zendesk.com/",
```

Comment:
This URL resolves to: https://support.xebialabs.com/hc/en-us. Might be better to use that URL instead.